### PR TITLE
Ticket8 post comment

### PR DESCRIPTION
**Pull request by margaritisarg**

### CODE REVIEW

File: src/components/SingleReviewContainer/FormComment.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    function handleSubmit(event){
+        event.preventDefault();
+        setIsRefreshing(true)
+        api.postComment({username: "tickle122", body}).then(() => {
+            setIsError(false)
+            setIsRefreshing(false)
+        }).catch(() => {
+            setIsError(true)
+        })
+        setBody('')
```
Comment (Hard-Luck) [inline src/components/SingleReviewContainer/FormComment.jsx]: would you want to lose the comment if the post failed?

File: src/components/SingleReviewContainer/FormComment.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    if (isError) return <p>No internet connection. Can't add a comment</p>
+    if (isRefreshing) return <p>Getting latest comment</p>
+    return(
+        <div>
+            <form className={formCommentCSS.form} onSubmit={handleSubmit}>
+                <section>
+                    <input type="text" placeholder="Comment"
```
Comment (Hard-Luck) [inline src/components/SingleReviewContainer/FormComment.jsx]: we should allow for multi-line comments, look into html text-area

**Review by Hard-Luck — COMMENTED** [review]
A couple of changes required and then you can merge

**Review by HeatherEver — APPROVED** [review]
Merge away!

**Pull request merged**